### ISSUE TITLE
Unexpected $ when using fragments with gql tag

### ISSUE THREAD

**Issue by FredyC** [dotansimha/graphql-code-generator#945]
**Describe the bug**
This is something that was working on 0.9.2. After upgrading to 0.14.4 (+ migrated configuration) I am getting `Syntax Error: Unexpected $`. It feels like it's more related to some transpilation issue since this just a literal template string. Or perhaps lack of it? Is there some transpilation happening at all?

**To Reproduce**

```js
export const GeoPointFragment = gql`
  fragment FGeoPoint on GeoPoint {
    latitude
    longitude
  }
`

export const AddressFragment = gql`
  fragment GAddress on Address {
    id
    location {
      ...FGeoPoint
    }
  }
  ${GeoPointFragment}
`
```

**Configuration**

```js
  await generate(
    {
      schema: jsonPath,
      overwrite: true,
      documents: ['./src/**/*graphql.ts'],
      generates: {
        [pathUtils.join(process.cwd(), 'src', 'graph', 'types.ts')]: {
          plugins: ['typescript-common', 'typescript-client'],
          config: {
            noNamespaces: true,
          },
        },
      },
    },
    true,
  )
```

**Environment:**
 - OS: Windows 10
 - Node: 10.12.0

```json
    "graphql-code-generator": "~0.14.4",
    "graphql-codegen-typescript-common":"~0.14.4",
    "graphql-codegen-typescript-client": "~0.14.4",
    "typescript": "^3.1.6"
```

It's Create-React-App project, but that's probably not related.

**Comment by FredyC** [comment]
Ok, I've found the culprit and opened an issue in upstream repo. You can keep this open to track progress or close it. Up to you.

**Comment by FredyC** [comment]
Fixed in #944
